llm_toolkit/prompts.py

```diff
@@ -144,8 +144,10 @@ def save(self, location: str) -> None:
 
   def append(self, text: str) -> None:
     """Appends to the formatted prompt."""
-    # A placeholder for now.
-    del text
+    self._prompt.append({
+        'role': 'system',
```

Comment:
why are we using `system` here as opposed to `user` or `assistant`?

Reply:
Have fixed, only the first append will be in role system, otherwise the appending text will be in user role to distinguish problem from priming.